src/orca-loader/Hyper.res

```diff
@@ -325,6 +325,79 @@ let make = (publishableKey, options: option<Js.Json.t>, analyticsInfo: option<Js
           addSmartEventListener("message", handleMessage, "onSubmit")
         })
       }
+
+      let confirmOneClickPayment = (payload, result: bool) => {
+        let confirmParams =
```

Comment:
There is repetitive code in `confirmOneClickPayment` and `confirmPayment`. Could you please move the common code outside

Reply:
Resolved this